src/cephadm/cephadm

```diff
@@ -4513,6 +4526,11 @@ def _get_parser():
         action='store_true',
         help='Do not automatically provision monitoring stack (prometheus, grafana, alertmanager, node-exporter)')
 
+    parser_bootstrap.add_argument(
+        '--shared_mgr_folder',
+        metavar='MGR_SOURCE_FOLDER',
+        help='Development mode. "/usr/share/ceph/mgr" in containers is a volume mapped to the manager modules source folder')
```

Comment:
what about changing the base location to be the ceph source root directory? that would enable us the use the same flag name for different containers as well? (i.e. ceph-volume)

Reply:
Is a good idea!. I will check to see if we have any kind of unexpected behaviour, and if it is ok I will make the update.